pyaedt/modules/DesignXPloration.py

```diff
@@ -406,6 +417,7 @@ def add_goal(self, calculation="", calculation_value="", calculation_type="discr
     def parent(self):
         """Parent object."""
         return self._parent
+    
```

Comment:
Thanks for adding this.  Nice to see you can contribute to coding style changes. Every little bit helps.